frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -77,4 +104,71 @@ export class VirtualMachine extends BaseVirtualMachine {
       await this.deleteNodeSelector(key);
     });
   }
+
+  async start(waitForAction = true) {
+    await this.action(VM_ACTION.Start, waitForAction);
+  }
+
+  async restart(waitForAction = true) {
+    await this.action(VM_ACTION.Restart, waitForAction);
+  }
+
+  async stop(waitForAction = true) {
+    await this.action(VM_ACTION.Stop, waitForAction);
+  }
+
+  async migrate(waitForAction = true) {
+    await this.action(VM_ACTION.Migrate, waitForAction);
+  }
+
+  async delete(waitForAction = true) {
+    await this.action(VM_ACTION.Delete, waitForAction);
+  }
+
+  async clone(name?: string, namespace?: string): Promise<VirtualMachine> {
```

Comment:
@suomiy wdyt?

Reply:
usage is https://github.com/openshift/console/pull/5823/files#diff-b7e6e8ec3576408c2fcd89598db464cdR240

Reply:
looks great and user friendly
One last touch could be a usage of builder here and using setName/setNamespace instead of plain object.

Reply:
I wanted to use the VMBuilder, but it creates dependency cycle....
That's why I did it this way

Reply:
ok np. Looks nice as it is